<launch>

    <node pkg="ti_viz_semseg" type="viz_semseg" name="viz_semseg" output="screen" args="" required="true">

        <param name="width"         value="1280"/>
        <param name="height"        value="720"/>
        <param name="tensor_width"  value="768"/>
        <param name="tensor_height" value="432"/>

        <!-- Input recitified image topic name to subscribe to -->
        <param name = "rectified_image_topic"    value = "camera/right/image_rect_mono"/>

        <!-- Input CNN tensor topic name to subscribe to -->
        <param name = "semseg_cnn_tensor_topic"  value = "semseg_cnn/tensor"/>

        <!-- Output semantic segmentation image topic name to publish to -->
        <param name = "semseg_cnn_image_topic"   value = "semseg_cnn/out_image"/>

    </node>

</launch>
